### ISSUE TITLE
Add option to drag and drop reorder of playlists

### ISSUE THREAD

**Issue by ANewDawn** [Alaanor/beatlist#47]
Be nice to be able to arrange songs in playlist by dragging up and down

**Comment by Alaanor** [comment]
Yes it's in the roadmap. But I'm unsure when I'll implement it. But that's definitively a good idea.
